### PR TITLE
Added bzr version 2.7.0

### PR DESCRIPTION
**Pull request by 3eka**
One from http://sfe.opencsw.org depends on Python 2.6 (we do not using anymore)

### CODE REVIEW

File: components/developer/bzr/bzr-27.p5m [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# <transform file path=usr.*/man/.+ -> default mangler.man.stability uncommitted>
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
```
Comment (pyhalov) [inline components/developer/bzr/bzr-27.p5m:25]: empty manifest????

**Comment by 3eka** [comment]
Argh, 
committed from wrong branch... Now, I have problem:
`$ gmake clobber`
`$ gmake install`
`$ gmake sample-manifest`

all went fine, so copied build/manifest-i386-generated.p5m to bzr-27.p5m and then
`$ gmake pre-publish`
failed:
```/usr/bin/pkgdepend resolve  -m /export/home/zeka/oi-userland/components/developer/bzr/build/manifest-i386-bzr-27.depend
/export/home/zeka/oi-userland/components/developer/bzr/build/manifest-i386-bzr-27.depend has unresolved dependency '
    depend type=require fmri=__TBD pkg.debug.depend.reason=usr/bin/bzr \
        pkg.debug.depend.type=python \
        pkg.debug.depend.file=64/profile_imports.so \
        pkg.debug.depend.file=64/profile_importsmodule.so \
        pkg.debug.depend.file=profile_imports.py \
        pkg.debug.depend.file=profile_imports.pyc \
        pkg.debug.depend.file=profile_imports.pyo \
        pkg.debug.depend.file=profile_imports.so \
        pkg.debug.depend.file=profile_imports/__init__.py \
        pkg.debug.depend.file=profile_importsmodule.so \
        pkg.debug.depend.path=usr/bin \
        pkg.debug.depend.path=usr/lib/python2.7 \
        pkg.debug.depend.path=usr/lib/python2.7/lib-dynload \
        pkg.debug.depend.path=usr/lib/python2.7/lib-old \
        pkg.debug.depend.path=usr/lib/python2.7/lib-tk \
        pkg.debug.depend.path=usr/lib/python2.7/plat-sunos5 \
        pkg.debug.depend.path=usr/lib/python2.7/site-packages \
        pkg.debug.depend.path=usr/lib/python2.7/vendor-packages \
        pkg.debug.depend.path=usr/lib/python2.7/vendor-packages/gtk-2.0 \
        pkg.debug.depend.path=usr/lib/python27.zip'.
gmake: *** [/export/home/zeka/oi-userland/make-rules/ips.mk:399: /export/home/zeka/oi-userland/components/developer/bzr/build/.resolved-i386] Error 1
```
any idea?

Regards.

**Comment by pyhalov** [comment]
It looks for profile_imports python module. If you sure that it's soft dependency, you can set pkg.depend.bypass-generate attribute on the file.

**Comment by pyhalov** [comment]
Should it be updated/patched or does it have fix for CVE-2017-14176 ?

**Comment by 3eka** [comment]
No patch. Will check...

**Comment by 3eka** [comment]
CVE-2017-14176 is handled on https://bugs.launchpad.net/bzr/%2Bbug/1710979 - but I cannot download patch from https://code.launchpad.net/~jelmer/brz/fix-ssh-sec ("Unauthorized") ....

Where else I can search for fix? Official site does not mention it at all...

Thanks

**Comment by pyhalov** [comment]
Try looking  at debian patches - https://launchpad.net/ubuntu/+archive/primary/+files/bzr_2.7.0+bzr6619-7ubuntu0.1.debian.tar.xz

**Comment by 3eka** [comment]
OK, found... Thanks.

**Comment by pyhalov** [comment]
The manifest is still empty. Please, use PYV manifest, even when you deliver package only for python 2.7. This will make further work easier.

**Comment by 3eka** [comment]
OK, will do... Will try to get this manifest filled in. Sorry for ignorance...

**Comment by 3eka** [comment]
I still have problems: bzr won't build in 32 bit, and import-profiler IS 32 bit version.
Need more time to make decision how to proceede...

**Comment by pyhalov** [comment]
import_profiler is arch-independent, it doesn't contain any dlls

**Comment by 3eka** [comment]
I saw this, but still: 
```
/export/home/zeka/oi-userland/components/developer/bzr/build/manifest-i386-bzr-27.depend has unresolved dependency '
    depend type=require fmri=__TBD pkg.debug.depend.reason=usr/bin/bzr \
        pkg.debug.depend.type=python \
        pkg.debug.depend.file=64/profile_imports.so \
        pkg.debug.depend.file=64/profile_importsmodule.so \
        pkg.debug.depend.file=profile_imports.py \
        pkg.debug.depend.file=profile_imports.pyc \
        pkg.debug.depend.file=profile_imports.pyo \
        pkg.debug.depend.file=profile_imports.so \
        pkg.debug.depend.file=profile_imports/__init__.py \
        pkg.debug.depend.file=profile_importsmodule.so \
        pkg.debug.depend.path=usr/bin \
        pkg.debug.depend.path=usr/lib/python2.7 \
        pkg.debug.depend.path=usr/lib/python2.7/lib-dynload \
        pkg.debug.depend.path=usr/lib/python2.7/lib-old \
        pkg.debug.depend.path=usr/lib/python2.7/lib-tk \
        pkg.debug.depend.path=usr/lib/python2.7/plat-sunos5 \
        pkg.debug.depend.path=usr/lib/python2.7/site-packages \
        pkg.debug.depend.path=usr/lib/python2.7/vendor-packages \
        pkg.debug.depend.path=usr/lib/python2.7/vendor-packages/gtk-2.0 \
        pkg.debug.depend.path=usr/lib/python27.zip'.
gmake: *** [/export/home/zeka/oi-userland/make-rules/ips.mk:403: /export/home/zeka/oi-userland/components/developer/bzr/build/.resolved-i386] Error 1
```
Why it insists then on:
```
        pkg.debug.depend.file=64/profile_imports.so \
        pkg.debug.depend.file=64/profile_importsmodule.so \
```
Is there any way to avoid that? Sorry, I am not experienced enough to get solution on my own..

Regards.

**Pull request closed without merging**